src/main/scala/wdlTools/types/Util.scala

```diff
@@ -73,7 +75,7 @@ object Util {
         s"WorkflowSig($name, input={$inputs}, outputs={$outputs})"
 
       // The type of a call to a task or a workflow.
-      case T_Call(name, output: Map[String, T]) =>
+      case T_CallDef(name, output: Map[String, T]) =>
```

Comment:
I am fine with `T_TaskDef and `T_WorkflowDef` they convey the meaning that we are defining something new.  However, I don't think `T_CallDef` is such a great name. I prefer leaving it `T_Call`.